src/components/SearchBar.vue

```diff
@@ -311,6 +308,29 @@ export default {
 
 <style lang="scss" scoped>
 .search-bar {
+  .foo {
```

Comment:
The foo class seems to be used (and therefore its children)

Reply:
Oups sorry!